karma_chameleon/main.py

```diff
@@ -127,9 +127,12 @@ def handle_karma_command(ack: Ack, say: Say, command: dict) -> None:
     }
 
     if event_type == "++":
-        say(app.increment_karma(msg))
+        rsp = app.increment_karma(msg)
```

Comment:
Any reason why this was split out into two calls? I don't necessarily have a problem with it but I'm just curious.